.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+ 
+branches:
+  only:
+    - master
+ 
+env:
+  global:
+    - URL="https://w3c.github.io/did-core/ECHIDNA"
+    - DECISION="https://www.w3.org/2019/did-wg/Meetings/Minutes/2019-10-22-did#resolution1"
+    - secure: "XB1q1xJNjGnW77S9dQTXDqlZNLUDYLQeotLMnrdIxTx9uwRrVWTCDZ6WmUKjJ3dWKeoU9YF2aL8V9AK8DejDsoYp9NVBk4Vyym6R3o62v2k4lClUo9sDuNwSTBHw6mmIdzXL4ihjwyGaIhkyGWb0EFNTPu31m1PEAoE0kjeLM2aRXYOZD0p2Zo4oStz+uHNk+6Za8aLMLq+zP/XYksWuNr0FqdHFddsdFhSMzEEVpjnzWnP+ix5+mOPsoYJgPJ0S2NSa7q+wHCtI8Dr0pyK1CiMdnGR18z3W67MftaLiLiRSVj7JqxninFRViu/4ZIyqG2tr34Yo2SC3E4CCECNYQ/XktKeu02q6pS87XRuHCgmrBcMkmDM6rFwicBBceq49zXFp46GPbrJuVQCMM8tVgMRzu5llsDyyQRr2QHbOSfxN15lmacygEegjHBro9iWv+181FGeXxU42ffL8RTq6JH3ZW256sFZVNszdnUo/EXM5x4ZKXVLFd7WmK32bR+2/ZO1VhMyjTSCIGX7qi0FKj373TwfuJQQG/IFOueR+K56DaY7+v+qFQq48ueyuZZvLOp665L5Z0oTR1BAu3NONQfICHl9BoA5rEywNjobuQ2+ZpgE2hAiA9l58/8zlklG/JtoaEq3xIZq7/O2r7lmq8yuFVsJKmDVRDjojo7831zE="
```

Comment:
Err, I don't know how this file works, but it looks like we're going to commit some kind of secret to the repository? Can you use this secret to do damage to W3C servers?

Reply:
I guess my question is... can I take this "secure" value and attack anyone's servers with it?

Reply:
To be honest, I do not know the details; maybe @deniak knows. 

What I know: this 'secure' is, I believe, like a signature of a data which combines a secure token I have got from github as an administrator, plus a secure token that I have received for this specific repository and echidna process, all generated by travis while I log in _in that specific repository_. If and when we set up the same echidna process for, say, the use cases (which I presume we will) then I will have to ask for a separate token for that repository+echidna and regenerate a 'secure' by being logged into that repository, yielding a different 'secure'. 

Bottom line is: I do not think so...

Reply:
The `secure` variable is the [encrypted token](https://github.com/w3c/echidna/wiki/Token-creation) Echidna uses to make sure the request is safe. Only travis can decrypt it.
Also, the token is linked to a specific GitHub repository, so even if someone can get the token, he will only be able to publish what's in that repository.

Reply:
Thank you @deniak, that answers my question. We're good to go here.